src/draw/sw/lv_draw_sw_private.h

```diff
@@ -38,6 +38,7 @@ struct lv_draw_sw_unit_t {
     lv_thread_t thread;
     volatile bool inited;
     volatile bool exit_status;
+    volatile bool wait_for_finish;
```

Comment:
I see why you put this member in the sw_unit struct: because `typedef lv_draw_sw_unit_t lv_draw_vglite_unit_t`. Maybe `lv_draw_vglite_unit_t` should be its own struct now?

Reply:
I put it here because some other units might use it.

Reply:
I agree to move it to VG-Lite rather. If others will need it they can also create their own types.

Reply:
Ok, do you want to change it now or later on in another PR after the 9.2 gets released?

Reply:
Yes, please change it now. Otherwise people might start using it and it will be more difficult to change after the release.

Reply:
Done.